### PR TITLE
Clean up for Creator and Business models

### PR DESCRIPTION
**Pull request by clacayetano**
This PR has the following updates:
- add email for FB scope (also removed unnecessary permissions, **for now**)
- rename User model to Creator
- fix env var problems when deploying to Heroku
- do not use https createServer if in production
- use Facebook ID as the basis of Creator record creation, not email

### CODE REVIEW

File: backend/data/models/Creator.js [hunk 1]
```diff
@@ -56,4 +54,11 @@ const Creator = connection.define(
   }
 );
 
+Creator.authenticate = function (user) {
+  return Creator.findOrCreate({
+    where: { facebookId: user.facebookId },
+    defaults: user
+  });
+};
+
```
Comment (StellarApp) [inline backend/data/models/Creator.js:63]: moving this findOrCreate function to the model looks more clean. Thank you!

File: frontend/src/components/Login.jsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ class Login extends Component {
         <div id="creatorLogin">
           <h2>For Creators:</h2>
           <FacebookLogin
-            appId="507675923424391"
+            appId={process.env.FB_APP_ID}
```
Comment (StellarApp) [inline frontend/src/components/Login.jsx:45]: So, does it mean we use process.env file in the frontend when it is deployed?  How did you make heroku read .env file?

**Review by StellarApp — APPROVED** [review]

**Comment by StellarApp** [comment]
It looks good! Thank you for your great work! I left one comment regarding deploying to Heroku with .env file. I will appreciate if you can answer my question.

File: frontend/src/components/Login.jsx (hunk 2 above)
Comment (clacayetano) [reply frontend/src/components/Login.jsx:45]: Hi @StellarApp, by adding [`systemvars: true`](https://github.com/StellarApp/capstone-influencer-search/blob/9ca48a52b1ad2cb7ca81e6104d1c3878304e5ccc/webpack.config.js#L26) in our webpack-config.js file, it allowed webpack to see the environment variables we defined in Heroku through the `Config Vars`.

**Pull request merged**